docs/diagrams/SummariseClassDiagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+skinparam backgroundColor #ffffff
+skinparam classAttributeIconSize 0
+hide circle
+
+abstract class "{abstract}\nCommand" {
+    + {abstract} execute(model : Model) : CommandResult
+}
+
+interface Model <<interface>> {
+    + addPerson(Person person) : void
+    + hasPerson(Person person) : boolean
```

Comment:
Similiar to the class diagram for pie chart window

For function that returns nothing, I think you can omit the void part
```suggestion
    + addPerson(Person)
    + hasPerson(Person) : boolean
```

Reply:
I agree with this change.

Reply:
Thanks for the suggestions. I have made the amendments.